services/terraform/eks-provision.tf

```diff
@@ -9,7 +9,10 @@
 #   without spilling secrets into the logs comes from:
 #   https://medium.com/citihub/a-more-secure-way-to-call-kubectl-from-terraform-1052adf37af8
 
-output "cluster_id" { value = module.eks.cluster_id }
+variable labels {
```

Comment:
Why labels and not tags? They look more like AWS tags to me, but there's some explanation missing about why you want to include the k8s labels as aws tags.

Reply:
I'm following [the nomenclature in the spec](https://github.com/cloudfoundry-incubator/cloud-service-broker/blob/master/docs/brokerpak-specification.md#provision)... The terraform here is mapping here from the "label" inputs in the OSBAPI domain space to the "tags" outputs in the AWS domain space.

The explanation for why to do this: First, this bit from the PR description:
> This will help us create a findable "solr" k8s cluster when the broker is deployed. Then the Terraform for the Solr brokerpak can refer to it using a data source and use that cluster by default, unless k8s access details are explicitly provided.

Second: The tags ensure we can properly [attribute the cost of any AWS resources back to the specific org, space, and instance that caused the broker to bring them into existence](https://github.com/cloudfoundry-incubator/cloud-service-broker/blob/master/docs/billing.md)... Not necessarily important given it's just us using this now, but it will be critical if TTS Tech Portfolio takes the broker over. Also, it's "what you're expected to do", cost-control-wise... That's why the labels are passed through OSBAPI.